src/main/scala/nl.knaw.dans.easy.springfield/CreateSpringfieldActions.scala

```diff
@@ -68,7 +68,7 @@ trait CreateSpringfieldActions {
   }
 
   def createSpringfieldActions(videos: Seq[Video]): Try[Elem] = Try {
-    <actions>
+    <actions xmlns="springfield-actions">
```

Comment:
@stephanHogenboom Have a look at this: https://www.w3schools.com/XML/xml_namespaces.asp
and then at this: https://easy.dans.knaw.nl/schemas/extern/springfield-actions.xsd.

Reply:
Never mind, I see now that the namespace identifier of springfield actions indeed is not a URI. @lindareijnhoudt : did you create it or know who did?

Reply:
i did. i've changed it to a URI
https://github.com/DANS-KNAW/easy-schema/pull/69

Reply:
https://github.com/DANS-KNAW/easy-ingest-flow/pull/108